app/components/AnnouncementInLine/index.tsx

```diff
@@ -13,28 +13,28 @@ type Props = {
 };
 
 const AnnouncementInLine = ({ event, meeting, group }: Props) => {
+  const navigate = useNavigate();
   const actionGrant = useAppSelector((state) => state.allowed.announcements);
 
   if (!actionGrant) {
     return null;
   }
 
   return (
-    <Link
-      to="/announcements"
-      state={
-        {
-          event,
-          meeting,
-          group,
-        } as AnnouncementCreateLocationState
-      }
+    <LinkButton
+      onPress={() => {
+        navigate('/announcements', {
+          state: {
+            event,
+            meeting,
+            group,
+          } as AnnouncementCreateLocationState,
+        });
+      }}
```

Comment:
Just an idea, but could we move this logic to the `LinkButton`-definition so that you could use it like this in this case:

```jsx
<LinkButton href='/announcements' state={ { event, meeting, group} as AnnouncementCreateLocationState }>
```

I think this could be a bit more clean and `react-router-dom` is already a dependency for `lego-bricks`, although the current solution is also okay.

Reply:
➕

Reply:
Yes, I didn't realize react-router was a lego-bricks dependency. It that case it might even be possible to get rid of the RouterContext thing

Reply:
I have changed my mind here. I don't think we should have `react-router-dom` as a dependency for `lego-bricks` as not all our projects use `react-router`, f.ex. kvittering uses `next` instead. I believe this means we cannot use `lego-bricks` for that project. 
But I do agree that this is a bit ugly, perhaps we could add this functionality in a separate `LinkButton`-component in `lego-webapp`

Reply:
Nevermind, I figured out a way to make it an optional peer dependency:)